doc/uftrace-info.md

```diff
@@ -78,6 +78,32 @@ To see the symbol table, one can use the `--symbols` option.
     [ 4] mcount (0x400570) size: 16
     [ 5] __cxa_atexit (0x400580) size: 16
 
+
+You can get symbol information from ELF binary directly without trace data that crafted by uftrace with record option.
```

Comment:
I think "that crafted by ..." part can be omitted.

Reply:
@namhyung if you think so...